inginious/backend/backend.py

```diff
@@ -174,12 +173,8 @@ async def handle_client_get_queue(self, client_addr, _: ClientGetQueue):
 
         #jobs_waiting: a list of tuples in the form
         #(job_id, is_current_client_job, info, launcher, max_time)
-        jobs_waiting = list()
-
-        for job in self._waiting_jobs.values():
-            if isinstance(job.msg, ClientNewJob):
```

Comment:
this check disappeared. Expected?

Reply:
Since the message is no more supposed to be replaced by ``None``, the ``job.msg`` type in ``WaitingJob``tuple is always supposed to be a ``ClientNewJob``.